cni/network/plugin/main.go

```diff
@@ -170,6 +171,7 @@ func main() {
 	upTime, err := platform.GetLastRebootTime()
 	if err == nil {
 		cniReport.VMUptime = upTime.Format("2006-01-02 15:04:05")
+		return
```

Comment:
why we return here. cni shouldnt exit here if it fails to get last reboot time. recently we removed cni dependency on reboot time.